src/libServer/GetWorkServer.cpp

```diff
@@ -34,10 +34,22 @@ using namespace jsonrpc;
 
 static ethash_mining_result_t FAIL_RESULT = {"", "", 0, false};
 
+namespace {
+
+std::shared_ptr<rpc::APIServer> GetServerConnector() {
+  rpc::APIServer::Options options;
+  options.threadPoolName = "GetWork";
+  options.numThreads = 2;
```

Comment:
Any specific reason why it's 2? comment?

Reply:
I believe the question here was to just put a comment in the code, not to make it a magic constant, @art-gor